### DISCUSSION TITLE
Can't load different profiles to multiple controllers of same type with auto profile

### DISCUSSION THREAD

**Discussion by sheendr** [AntiMicroX/antimicrox#674] [Q&A]
I have two x-box wireless controllers and a logitech wired controller.  I can use auto profile to detect a mame game and load a profile for player 1 to the logitech controller and another profile for player 2 to one of the x-box controllers.  My problem is I can't figure out how to load a different profile to each of the two x-box controllers with auto profile. I believe they have a different unique IDs, but in the "devices" list that is at the bottom of the auto profile dialog box both x box controllers have the same name (x box series x controller).  There are two entries but the same name and I couldn't get different profiles assigned to each one.  Is there a way to do this I am missing?

I've had challenges learning the interface but this is the first issue that has me stymied.  Thoughts?

**Comment by sheendr** [comment]
Forgot to mention, I'm running antimicroX version 3.1.4 with  Ubuntu version 22.04.1 LTS.

**Reply by pktiuk** [reply]
You use outdated version (at least a year old). I don't recommend using it.
